testsuite/integration/src/test/xslt/setupLegacyIiopSslSupport.xsl

```diff
@@ -44,21 +62,4 @@
             </xsl:otherwise>
         </xsl:choose>
     </xsl:template>
-
-    <!-- Enable SSL support in Jacorb configuration -->
-    <xsl:template match="//*[local-name()='subsystem' and starts-with(namespace-uri(), $jacorbNS)]">
```

Comment:
Same as above, new template was added above that one to properly setup SSL and pass the new validations.